server/api/costumes.ts

```diff
@@ -17,5 +17,34 @@ costumesRouter.get("/", async (req, res, next): Promise<void> => {
     }
 })
 
+// POST /api/costumes
+
+costumesRouter.post('/', async (req, res, next): Promise<void> => {
+    try {
+        const {
+            name, 
+            category, 
+            gender, 
+            size, 
+            type, 
+            stockCount,
+            price 
+        } = req.body;
+        const costume = await createCostume(
+            pool, {
+                name, 
+                category, 
+                gender,
+                size,
+                type,
+                stockCount,
+                price
+            });
```

Comment:
Wouldn't this work, since they have the same keys?
```suggestion
        const costume = await createCostume(pool, req.body);
```

Reply:
I suppose it wouldn't if there were more properties in `req.body` that weren't destructured. But I don't think that is the case, so, yeah, I'll give that a try. Thanks!

Reply:
It worked!